src/lib/holocene/select/option.svelte

```diff
@@ -5,6 +5,7 @@
     description?: string;
     disabled?: boolean;
     class?: string;
+    ['data-testid']?: string;
```

Comment:
Wouldn't this just be 'data-testid'?: string;

Reply:
Or just not have it here if it's used on $$restProps

Reply:
I think I wanna leave it because it's something I want to be explicitly in the definition, didn't know how to access a variable with a - in int without restProps tho

Reply:
🪃 🪃 🪃 🪃 🪃